backend/app.go

```diff
@@ -169,17 +174,17 @@ func (a *WalletApplication) initDirectoryStructure() error {
 
 // initMainnetConnection populates the WalletApplication struct with mainnet data
 func (a *WalletApplication) initMainnetConnection() {
-	a.Network.URL = "http://lb.constellationnetwork.io:9000" // Temp
+	a.Network.URL = MainnetLoadBalancerURL // Temp
 
 	a.Network.Handles.Send = "/send"
 	a.Network.Handles.Transaction = "/transaction"
 	a.Network.Handles.Balance = "/address/"
 
-	a.Network.BlockExplorer.URL = "https://xju69fets2.execute-api.us-west-1.amazonaws.com/cl-block-explorer-mainnet"
+	a.Network.BlockExplorer.URL = MainnetBlockExplorerURL
 	a.Network.BlockExplorer.Handles.Transactions = "/transactions/"
```

Comment:
Update these endpoints to match BE V2 API spec.

Reply:
Got it.